atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -420,6 +421,7 @@ def _get_openshift_session(self, kwargs):
 
     def get_cluster_info(self, cluster, platform):
         kwargs = deepcopy(self.config_kwargs)
+        kwargs['reactor_config_yaml'] = yaml.dump(self.reactor_config)
```

Comment:
Can we only set this if a non-empty `self.reactor_config` is returned by `get_config` ?

This might need a new method in `ReactorConfig` class to determine if config is empty. Essentially a comparison of `self.conf` with `DEFAULT_CONFIG`.

Reply:
Also, I don't think we need to serialize `self.reactor_config` here. That should probably be done by osbs-client when processing this value and stuffing it in an environment variable. Thoughts?

Reply:
fixed